### PR TITLE
added user type check before creating new asset

### PR DESCRIPTION
**Pull request by kshitijv256**
## Proposed Changes

- Added user type check before creating new asset

### Associated Issue

- https://github.com/coronasafe/care_fe/issues/7081

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Review by sainak — CHANGES_REQUESTED** [review]
use dry permissions for this validation
https://github.com/dbkaplan/dry-rest-permissions

**Review by rithviknishad — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

File: care/facility/api/serializers/asset.py [hunk 1]
```diff
@@ -213,6 +214,7 @@ def validate(self, attrs):
         return super().validate(attrs)
 
     def create(self, validated_data):
+        user = self.context["request"].user
```
Comment (sainak) [inline care/facility/api/serializers/asset.py:217]: where is this being used?

**Comment by sainak** [comment]
add test cases

**Comment by kshitijv256** [comment]
> add test cases

@sainak 
I am unable to create a test case as different user type is needed, can you give some idea how

**Comment by rithviknishad** [comment]
@kshitijv256 example: https://github.com/coronasafe/care/pull/1866/files#diff-6e69c1bec5735c00ad8c3637de1e270b87399c4a6958c4496a9d145e3025d333R90

**Pull request closed without merging**